sycl/test/basic_tests/accessor/host_acc_opt.cpp

```diff
@@ -9,7 +9,6 @@
 // CHECK: define {{.*}}foo{{.*}} {
 // CHECK-NOT: call
 // CHECK-NOT: invoke
-// CHECK: vector.body:
 // CHECK-NOT: call
 // CHECK-NOT: invoke
```

Comment:
These aren't needed anymore.

Reply:
I see, will update in a separate PR. Thank you for noting.